src/Http/Http/src/Microsoft.AspNetCore.Http.csproj

```diff
@@ -18,6 +18,7 @@
     <Reference Include="Microsoft.AspNetCore.WebUtilities" />
     <Reference Include="Microsoft.Extensions.ObjectPool" />
     <Reference Include="Microsoft.Extensions.Options" />
+    <Reference Include="Microsoft.Extensions.Logging" />
```

Comment:
I hesitate to add this only for the pipe components that are going to move later. How will we remember to remove this?

Reply:
Pretty simple, once we move the types won’t exist :)

Reply:
How do we remember to remove the logging dependency?

Reply:
I mean all Microsoft.Extensions/ Microsoft.AspNetCore dependencies would be removed/not used.